testing/renderer-with-debug-ui.jsx

```diff
@@ -39,7 +39,6 @@ export const RendererWithDebugUI = ({
                         content={question.content}
                         images={question.images}
                         widgets={question.widgets}
-                        problemNum={0}
```

Comment:
Are you sure you want to delete this now that you're separating out the randomized tests?

Reply:
Good catch! Added back